docs/source/cluster_management/sweep.rst

```diff
@@ -45,34 +46,38 @@ Tunable Configuration Options
+Note that some of these parameters are just used as a hint. Sweep dynamically modifies these parameters according to successful or failed runs.
 
 .. csv-table::
-   :header: "AtlasDB Config", "CLI Option", "Default", "Description"
+   :header: "AtlasDB Runtime Config", "Endpoint Option", "Default", "Description"
    :widths: 20, 20, 40, 200
 
-   (DEPRECATED) ``sweepBatchSize``, (REMOVED), "100", "Maximum number of rows to sweep at once. Decrease this if sweep fails to complete (for example if the sweep job or the underlying KVS runs out of memory). Increasing it may improve sweep performance. This parameter should be replaced by ``--candidate-batch-hint``."
-   (DEPRECATED) ``sweepCellBatchSize``, (REMOVED), "10,000", "Maximum number of cells to sweep at once. Similar to ``sweepBatchSize`` but provides finer control if the row widths vary greatly."
-   ``sweepReadLimit``, ``--read-limit``, "1,000,000", "Target number of (cell, timestamp) pairs to examine in a single run."
-   ``sweepCandidateBatchHint``, ``--candidate-batch-hint``, "100", "Approximate number of candidate (cell, timestamp) pairs to load at once. Decrease this if sweep fails to complete (for example if the sweep job or the underlying KVS runs out of memory). Increasing it may improve sweep performance."
-   ``sweepDeleteBatchHint``, ``--delete-batch-hint``, "1,000", "Target number of (cell, timestamp) pairs to delete in a single batch."
-   ``sweepPauseMillis``, ``--sleep``, "5000 ms", "Wait time between row batches. Set this if you want to use less shared DB resources, for example if you run sweep during user-facing hours."
-   "``timestampsGetterBatchSize`` (Cassandra KVS only, see :ref:`Cassandra KVS config <cassandra-configuration>`)", "Only specified in config", "Fetch all columns", "Specify a limit on the maximum number of columns to fetch in a single database query. Set this to a number fewer than your number of columns if your Cassandra OOMs when attempting to run sweep with even a small row batch size. This parameter should be used when tuning Sweep for cells with many historical versions."
+   ``enabled``, "Only specified in config", "1,000", "Target number of (cell, timestamp) pairs to examine in a single run."
+   ``readLimit``, ``maxCellTsPairsToExamine``, "1,000", "Target number of (cell, timestamp) pairs to examine in a single run."
+   ``candidateBatchHint``, ``candidate-batch-hint``, "1", "Target number of candidate (cell, timestamp) pairs to load at once. Decrease this if sweep fails to complete (for example if the sweep job or the underlying KVS runs out of memory). Increasing it may improve sweep performance."
+   ``deleteBatchHint``, ``delete-batch-hint``, "1,000", "Target number of (cell, timestamp) pairs to delete in a single batch. Decrease if sweep cannot progress pass a large row or a large cell. Increasing it may improve sweep performance."
```

Comment:
Do all these parameters appear in https://github.com/palantir/atlasdb/blob/develop/docs/source/cluster_management/sweep/sweep-endpoints.rst ?

Also looks like there's disagreement on the casing used by Endpoints between this doc and the sweep-endpoints.rst doc.